app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -434,29 +391,14 @@ def destroy
     return head :unauthorized unless current_user.has_permission?(:can_edit_groups, role.group.id)
     role.update!(end_date: Date.today)
     RoleChangeMailer.notify_role_end(role, current_user).deliver_later
+    if role.user.delegate_roles.count == 0
```

Comment:
This should be almost identical to the `any_kind_of_delegate?` method in `user.rb`, right?!

Reply:
Yes, thanks. Done.